pages/services/nifi/0.5.0-1.9.2/troubleshooting/diagnostictools/index.md

```diff
@@ -0,0 +1,153 @@
+
+As of this writing, the best and fastest way to view and download logs is via the Mesos UI at `<dcos-url>/mesos`. On the Mesos front page you will see two lists: A list of currently running tasks, followed by a list of completed tasks (whether successful or failed).
+
+![mesos frontpage showing all tasks in the cluster](../../img/1_Logging_All_Tasks.png)
+
+Figure 1. - Mesos front page
+
+The Sandbox link for one of these tasks shows a list of files from within the task itself. For example, here’s a sandbox view of a nifi-0-node task from the above list:
+
+![inside task" width](../../img/2_Inside_Task.png)
+
+Figure 2. - Task file list
+
+If the task is based on a Docker image, this list will only show the contents of `/mnt/sandbox`, and not the rest of the filesystem. If you need to view filesystem contents outside of this directory, you will need to use `dcos task exec` or `nsenter` as described below under Running commands within containers.
+
+In the above task list(figure 1) there are multiple services installed, resulting in a pretty large list. The list can be filtered using the text box at the upper right, but there may be duplicate names across services. For example there are two instances of nifi and they’re each running a node-0. As the cluster grows, this confusion gets proportionally worse. We want to limit the task list to only the tasks that are relevant to the service being diagnosed. To do this, click “Frameworks” on the upper left to see a list of all the installed frameworks (mapping to our services):
+
+![active frameworks](../../img/3_Active_Frameworks.png)
+
+Figure 3. - Active Frameworks
+
+We then need to decide which framework to select from this list. This depends on what task we want to view.
+
+## Scheduler Logs
+
+If the issue is one of deployment or management, for example, a service is ‘stuck’ in initial deployment, or a task that previously went down is not being brought back at all, then the Scheduler logs will likely be the place to find out why.
+
+From Mesos’s perspective, the Scheduler is being run as a Marathon app. Therefore we should pick marathon from this list and then find our Scheduler in the list of tasks.
+
+![scheduler](../../img/4_Scheduler.png)
+
+Figure 4. - List of active tasks
+
+Scheduler logs can be found either via the main Mesos frontpage in small clusters (possibly using the filter box at the top right), or by navigating into the list of tasks registered against the marathon framework in large clusters. In SDK services, the Scheduler is typically given the same name as the service. For example a `nifi-dev` service’s Scheduler would be named `nifi-dev`. Use the Sandbox link to view the Sandbox portion of the Scheduler filesystem, which contains files named `stdout` and `stderr`. These files respectively receive the `stdout/stderr` output of the Scheduler process, and can be examined to see what the Scheduler is doing.
```

Comment:
"front page" is two words, not one.
Capitalize "Marathon".

Reply:
changed